src/scmdata/run.py

```diff
@@ -880,6 +881,9 @@ def filter(
             in a string column which contains ;class:`np.nan` will result in a
             :class:`TypeError`.
 
+        warn_if_empty
+            If ``True``, raise a warning if the result is empty.
```

Comment:
It's a log message, not an exception/warning

Reply:
better now?